thamos/cli.py

```diff
@@ -282,6 +326,79 @@ def _print_advise_justifications(
         else:
             click.echo("No justification was made for the recommended stack")
 
+    return None
+
+
+def _compute_metrics_scorecards(report) -> Optional[OrderedDict]:
+    """Aggregate OSSF Scorecards information from report and compute metrics."""
+    nb_dependencies = 0
+    scorecards_metrics = {}
+
+    if report["products"] and report["products"][0]["dependency_graph"]["nodes"]:
+        nb_dependencies = len(report["products"][0]["dependency_graph"]["nodes"])
+
+    for justification in report["products"][0]["justification"]:
+        if "scorecard" in justification["link"]:
+            if "tag" in justification:
+                message = _REPORT_TRANSLATION_TABLE_SCORECARD_TAGS[justification["tag"]]
+
+            # TO DO: Remove when corresponding prescriptions have been updated with tags
+            else:
+                message = justification["message"].strip()
+
+                if "Project " in message:
+                    message = message.replace("Project ", "projects ")
+
+                if " is " in message:
+                    message = message.replace(" is ", " are ")
+
+                if " does " in message:
+                    message = message.replace(" does ", " do ")
+
+                if " has " in message:
+                    message = message.replace(" has ", " have ")
+
+                if " runs " in message:
+                    message = message.replace(" runs ", " run ")
+
+                if " requires " in message:
+                    message = message.replace(" requires ", " require ")
+
+                if " uses " in message:
+                    message = message.replace(" uses ", " use ")
+
+                if " follows " in message:
+                    message = message.replace(" follows ", " follow ")
+
+                if " signs " in message:
+                    message = message.replace(" signs ", " sign ")
+
+            if " NOT " in message:
+                message = message.replace(" NOT ", " not ")
```

Comment:
can share what functionality is this for, it this to fix the grammer of the sentence ? 

also how about using a dict with these words and replacing them with a function looping over dict and message? is there any significant value to use `if` conditions here?

Reply:
I implemented this (dirty) workaround to correct the message grammar while we wait for more prescriptions to be generated using tags. When most prescriptions will contain metadata about the type of Scorecard they are based on in their tag, then we could remove this and leave only the logic that uses a dictionary I also implemented in this PR. Does this sound like a good temporary solution to you?

Reply:
ack, i think this clears my thoughts.
As we would like to have these grammar change till we wait on prescriptions to be generated using tags.
how about, make the grammer fix words into a dict solution, so that if we found something else needed to be correct just would need to be added to a dict instead of a new if statement.
just for easy of updates.

Example:
```
def fix_message(message, replace_words):
   for k,v in replace_words.items():
        message = message.replace(k,v)
   return message
   
replace_words = {"Projects":"project", "is":"are",...}
msg= fix_message(mesage,replace_wrods)
```

Wdyt?  i dont feel strongly about this, if you say it's temp, we can go with it.
Its just that we have too many if statement doing the same task.